treeherder/extract/extract_jobs.py

```diff
@@ -77,43 +91,25 @@ def run(self, force=False, restart=False, merge=False):
                 # get_ids = ConcatSQL(
                 #     (SQL_SELECT, sql_alias(quote_value(283890114), "id"))
                 # )
-                # get_ids = sql_query(
-                #     {
-                #         "from": "job",
-                #         "select": ["id"],
-                #         "where": {
-                #             "or": [
-                #                 {"gt": {"last_modified": parse(last_modified)}},
-                #                 {
-                #                     "and": [
-                #                         {"eq": {"last_modified": parse(last_modified)}},
-                #                         {"gt": {"id": job_id}},
-                #                     ]
-                #                 },
-                #             ]
-                #         },
-                #         "sort": ["last_modified", "id"],
-                #         "limit": settings.extractor.chunk_size,
-                #     }
-                # )
-
-                get_ids = SQL(str(
-                    (
-                        Job.objects.filter(
-                            Q(last_modified__gt=parse(last_modified).datetime)
-                            | (
-                                Q(last_modified=parse(last_modified).datetime)
-                                & Q(id__gt=job_id)
-                            )
-                        )
-                        .annotate()
-                        .values("id")
-                        .order_by("last_modified", "id")[
-                            : settings.extractor.chunk_size
-                        ]
-                    ).query
-                ))
-
+                get_ids = sql_query(
+                    {
+                        "from": "job",
+                        "select": ["id"],
+                        "where": {
+                            "or": [
+                                {"gt": {"last_modified": parse(last_modified)}},
+                                {
+                                    "and": [
+                                        {"eq": {"last_modified": parse(last_modified)}},
+                                        {"gt": {"id": job_id}},
+                                    ]
+                                },
+                            ]
+                        },
+                        "sort": ["last_modified", "id"],
+                        "limit": settings.extractor.chunk_size,
+                    }
+                )
```

Comment:
@armenzg  Since I can not seem to construct SQL using Django, I reverted to using another library

Reply:
Got it.